scalyr_agent/builtin_monitors/kubernetes_monitor.py

```diff
@@ -33,7 +33,7 @@
 import scalyr_agent.scalyr_logging as scalyr_logging
 from scalyr_agent.json_lib import JsonObject, ArrayOfStrings
 from scalyr_agent.monitor_utils.k8s import KubernetesApi, KubeletApi, KubeletApiException, K8sApiTemporaryError
-from scalyr_agent.monitor_utils.k8s import K8sApiPermanentError, DockerMetricFetcher, QualifiedName
+from scalyr_agent.monitor_utils.k8s import K8sApiPermanentError, DockerMetricFetcher, QualifiedName, ApiQueryOptions
```

Comment:
This appears to be related to the rate limit branch, not this feature?

It's not a major issue (assuming the code on your current ratelimit branch is the same), but it might cause merge conflicts so maybe best to leave this off this merge, or rebase it off your rate-limit branch instead of `option-b/core`?